### PR TITLE
Pool summary update

### PR DESCRIPTION
**Pull request by GusevTimofey**

### CODE REVIEW

File: modules/markets-api/src/main/scala/org/ergoplatform/dex/markets/services/FiatRates.scala [hunk 1]
```diff
@@ -1,29 +1,32 @@
...
 import scala.concurrent.duration._
 
-@derive(representableK)
-trait FiatRates[F[_]] {
+trait FiatRates[F[_], S[_]] {
 
   def rateOf(asset: AssetClass, units: FiatUnits): F[Option[BigDecimal]]
+
+  def run: S[Unit]
 }
```
Comment (oskin1) [inline modules/markets-api/src/main/scala/org/ergoplatform/dex/markets/services/FiatRates.scala:30]: Wtf? Why FiatRates a process and a service at the same time?

**Review by oskin1 — CHANGES_REQUESTED** [review]

File: modules/markets-api/src/main/scala/org/ergoplatform/dex/markets/services/FiatRates.scala (hunk 1 above)
Comment (GusevTimofey) [reply modules/markets-api/src/main/scala/org/ergoplatform/dex/markets/services/FiatRates.scala]: reworked

File: modules/markets-api/src/main/scala/org/ergoplatform/dex/markets/services/FiatRates.scala [hunk 2]
```diff
@@ -2,92 +2,55 @@ package org.ergoplatform.dex.markets.services
...
 
 object FiatRates {
 
   val ErgUsdPoolNft: TokenId =
     TokenId.fromStringUnsafe("011d3364de07e5a26f0c4eef0852cddb387039a921b7154ef3cab22c6eda887f")
 
-  val MemoTtl: FiniteDuration = 2.minutes
-
-  def make[I[_]: FlatMap: Sync, S[_]: Monad: Evals[*[_], F]: Catches, F[_]: Monad: Clock: Sync: Timer](implicit
+  def make[I[_]: FlatMap: Sync, F[_]: Monad: Clock: Sync: Timer](implicit
```
Comment (oskin1) [inline modules/markets-api/src/main/scala/org/ergoplatform/dex/markets/services/FiatRates.scala:30]: FlatMap + Sync? Monad + Sync?

File: modules/markets-api/src/main/scala/org/ergoplatform/dex/markets/modules/PriceSolver.scala [hunk 3]
```diff
@@ -49,15 +59,15 @@ object PriceSolver {
     implicit def representableK: RepresentableK[FiatPriceSolver] =
       tofu.higherKind.derived.genRepresentableK
 
-    def make[I[_]: Functor, F[_]: Monad](implicit
+    def make[I[_]: Functor, F[_]: Monad, S[_]](implicit
```
Comment (oskin1) [inline modules/markets-api/src/main/scala/org/ergoplatform/dex/markets/modules/PriceSolver.scala:62]: Why S[_] is here?

**Pull request merged**